features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given("I am on the landing page") do
+    visit root_path
+  end
+  
+And("I click on {string}") do |element|
+    click_on element
+end
+
+When("I fill in {string} with {string}") do |input, content|
+    fill_in input, with: content
+end
+
+Then("I should be on the Sign up page") do
```

Comment:
some of these steps need to be in an assertion step file

Reply:
ok